frontend/src/config/jordan/layers.json

```diff
@@ -2242,5 +2242,431 @@
+        "invert": false,
+        "key": [
+          "CHIRPS",
+          "r1h_dekad"
+        ]
+      },
+      {
+        "importance": 0.3,
+        "aggregation": "average",
+        "invert": true,
+        "key": [
+          "FAO",
+          "ET0"
+        ]
+      },
+      {
+        "importance": 0.3,
+        "aggregation": "last_dekad",
+        "invert": false,
+        "key": [
+          "AGERA5",
+          "soil_moist"
+        ]
+      }
+    ],
+    "legend": [
+      {
+        "value": 0,
+        "label": "0 - 0.1",
+        "color": "#672200"
+      },
+      {
+        "value": 0.1,
+        "label": "0.1 - 0.2",
+        "color": "#a93800"
+      },
+      {
+        "value": 0.2,
+        "label": "0.2 - 0.3",
+        "color": "#e59800"
+      },
+      {
+        "value": 0.3,
+        "label": "0.3 - 0.4",
+        "color": "#ffe769"
+      },
+      {
+        "value": 0.4,
+        "label": "0.4 - 0.5",
+        "color": "#f0f0f0"
+      },
+      {
+        "value": 0.5,
+        "label": "0.5 - 0.6",
+        "color": "#f0f0f0"
+      },
+      {
+        "value": 0.6,
+        "label": "0.6 - 0.7",
+        "color": "#a9c6d6"
+      },
+      {
+        "value": 0.7,
+        "label": "0.7 - 0.8",
+        "color": "#019ac4"
+      },
+      {
+        "value": 0.8,
+        "label": "0.8 - 0.9",
+        "color": "#014c73"
+      },
+      {
+        "value": 0.9,
+        "label": "0.9 - 1",
+        "color": "#014c73"
+      }
+    ],
+    "legend_text": "Composite Drought Index (water-based). Rainfall amount weight = 0.4; evapotranspiration weight = 0.3; soil moisture weight = 0.3",
+    "opacity": 0.7
+  },
+  "cdi_v3_aggregated_monthly": {
+    "title": "Monthly Combined Drought Index Aggregated (water-based)",
+    "type": "composite",
+    "period": "monthly",
+    "base_url": "https://hip-service.ovio.org/q_multi_aggregated",
+    "start_date": "1981-01-01",
+    "end_date": "2024-01-01",
+    "aggregation": "pixel",
+    "aggregate_by": "mean",
+    "admin_code": "admin3Pcod",
+    "aggregation_boundary_path": "data/jordan/jor_admbnda_adm3_jdos_merged.json",
+    "date_layer": "spi_1m",
+    "validity": {
+      "forward": 1,
+      "backward": 2,
+      "mode": "dekad"
+    },
+    "feature_info_props": {
+      "value": {
+        "type": "number",
+        "dataTitle": "Event name"
+      }
+    },
+    "input_layers": [
+      {
+        "importance": 0.4,
+        "aggregation": "last_dekad",
+        "invert": false,
+        "key": [
+          "CHIRPS",
+          "r1h_dekad"
+        ]
+      },
+      {
+        "importance": 0.3,
+        "aggregation": "average",
+        "invert": true,
+        "key": [
+          "FAO",
+          "ET0"
+        ]
+      },
+      {
+        "importance": 0.3,
+        "aggregation": "last_dekad",
+        "invert": false,
+        "key": [
+          "AGERA5",
+          "soil_moist"
+        ]
+      }
+    ],
+    "legend": [
```

Comment:
Can the legends be shared? Or are they all different?

Reply:
Yes, good point. I added a shared legend entry